recipes/cudatoolkit-dev/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "cudatoolkit-dev" %}
+{% set version = "9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ./
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  run:
+   - requests 
+   - conda 
+
+test:
+  commands:
+    - test -f $PREFIX/cudatoolkit-dev-post-install.py  # [linux]
```

Comment:
Could you also add a test to see if the files that should be installed as part of the build.py script are actually installed?